### ISSUE TITLE
magnet holes

### ISSUE THREAD

**Issue by bbrendon** [michaelgale/cq-gridfinity#7]
what size are the magnet holes for this? If they are 6.5 would it be possible to have an option to set it to something for press-fit?

Sorry, I couldn't get this project to run. Not sure what the issue is yet.

```
  File "/Users/x/p/x/venv/lib/python3.11/site-packages/cqgridfinity/gf_obj.py", line 28, in <module>
    from OCP.BRepMesh import BRepMesh_IncrementalMesh
ModuleNotFoundError: No module named 'OCP'
❯ pip install OCP
Requirement already satisfied: OCP in ./venv/lib/python3.11/site-packages (0.1.9)
```

**Comment by michaelgale** [comment]
@bbrendon Yes, currently the magnet hole diameter is a fixed value of 6.5 mm.  Unfortunately, there isn't a mechanism to override that value.  However, it is a modest change and I have just pushed a commit to implement this ability.  You can pass `hole_diam` as a keyword argument to `GridfinityBox` to override the default value of 6.5 mm.  

If you are getting that error from pip about OCP, it looks like you do not have a complete CadQuery installation working yet.  OCP is binary package that is best installed using conda.  I would consult the CadQuery documentation to make sure you've got that working first.  If you can do this at the python prompt:

```shell
$ python
>>> import cadquery
>>> cadquery.__version__
'2.4.0'
```
then you're good to go.

**Comment by michaelgale** [comment]
Closing.  Feature implemented.
